src/util/strencodings.h

```diff
@@ -68,8 +70,25 @@ std::string EncodeBase32(Span<const unsigned char> input, bool pad = true);
 std::string EncodeBase32(const std::string& str, bool pad = true);
 
 void SplitHostPort(std::string in, int& portOut, std::string& hostOut);
-int64_t atoi64(const std::string& str);
-int atoi(const std::string& str);
+
+template <typename T>
```

Comment:
Please add a comment what this function does (e.g.: replicate the exact behavior of `atoi`) and that new code should use the `Parse[U]IntXX` functions with error checking.

Reply:
Done! :)